rustls/src/crypto/aws_lc_rs/ticketer.rs

```diff
@@ -51,97 +56,124 @@ impl Ticketer {
 }
 
 fn make_ticket_generator() -> Result<Box<dyn ProducesTickets>, GetRandomFailed> {
-    let mut key = [0u8; 32];
-    SystemRandom::new()
-        .fill(&mut key)
-        .map_err(|_| GetRandomFailed)?;
-
-    let key = aead::UnboundKey::new(TICKETER_AEAD, &key).unwrap();
-
-    let mut key_name = [0u8; 16];
-    SystemRandom::new()
-        .fill(&mut key_name)
-        .map_err(|_| GetRandomFailed)?;
-
-    Ok(Box::new(AeadTicketer {
-        alg: TICKETER_AEAD,
-        key: aead::LessSafeKey::new(key),
-        key_name,
-        lifetime: 60 * 60 * 12,
-        maximum_ciphertext_len: AtomicUsize::new(0),
-    }))
+    // NOTE(XXX): Unconditionally mapping errors to `GetRandomFailed` here is slightly
```

Comment:
File a follow-up issue or PR with the breaking change label?

Reply:
Done: https://github.com/rustls/rustls/issues/2074